packages/frontend/tests/workers/crypto/nacl-test.ts

```diff
@@ -19,6 +19,13 @@ module('Workers | Crypto | nacl', function () {
     assert.ok(boxKeys.privateKey);
   });
 
+  test('generateSigningKeys | works', async function (assert) {
```

Comment:
Can you add a test that demonstrates signing / opening as well?

Note the encrypt/decrypt test a few lines below

Reply:
Those tests will be in #686

Reply:
sounds good!